content/millennium/r4/individuals/related-person.md

```diff
@@ -0,0 +1,107 @@
+---
+title: RelatedPerson | R4 API
+---
+
+# RelatedPerson
+
+* TOC
+{:toc}
+
+## Overview
+
+The RelatedPerson resource provides information about a person who is involved in the care of a patient but has no formal responsibility. RelatedPersons typically have a personal relationship with the patient such as a spouse, relative, friend, guardian, or attorney. RelatedPersons are commonly a source of information about a patient.
+
+The distinction between Practitioner and RelatedPerson is a practitioner cares for multiple patients on behalf of a healthcare facility where a RelatedPerson performs care tasks for a specific patient and is not associated with any healthcare facility.
+
+The following fields are returned if valued:
+
+* [RelatedPerson id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [RelatedPerson Identifier](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.identifier){:target="_blank"}
+* [Active](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.active){:target="_blank"} 
+* [Patient](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.patient){:target="_blank"}
+* [Relationship](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.relationship){:target="_blank"}
+* [Related person's name](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.name){:target="_blank"}
+* [Telecom information (may include phone and email)](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.telecom){:target="_blank"}
+* [Gender (administrative)](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.gender){:target="_blank"}
+* [Date of birth](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.birthDate){:target="_blank"}
+* [Address](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.address){:target="_blank"}
+* [Communication (preferred language)](https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.communication){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:related_person, :r4) %>
+
+## Search
+
+Search for RelatedPersons that meet supplied query parameters:
+
+    GET /RelatedPerson?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name         | Required?                                         | Type          | Description
+--------------|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------
```

Comment:
4 columns for the header and table body, but 3 for the table divider

Reply:
added extra table divider
https://github.com/cerner/fhir.cerner.com/pull/254/commits/6d1dc18c06362fc770aea2991e8e317ec263a54e#diff-b47a4cc256449f5aa96d433a01e1ae01R47